.github/ISSUE_TEMPLATE/feature_report.yml

```diff
@@ -0,0 +1,63 @@
+name: "💡 Feature Request"
+description: Create a new ticket for a new feature request
+title: "[REQUEST] - <title>"
+labels: [
+  "question"
+]
+body:
+  - type: input
```

Comment:
can remove lines 8-31. I want this to include a feature someone may want in the future but hasn't necessarily implemented. 

I'm okay with keeping the sections:
1. Summary
2. Basic Example
3. Drawbacks

Reply:
Let me know if you have questions.